opencart 1.5.6.4/upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -214,12 +214,112 @@
               </td>
           </table>
         </div>
+        <table id="module" class="list">
+          <thead>
+            <tr>
+              <td class="left"><?php echo $entry_title_layout; ?></td>
+              <td class="left"><?php echo $entry_title_position; ?></td>
+              <td class="left"><?php echo $entry_title_status; ?></td>
+              <td class="right"><?php echo $entry_title_sort_order; ?></td>
+              <td></td>
+            </tr>
+          </thead>
+          <?php $module_row = 0; ?>
+          <?php foreach ($modules as $module) { ?>
+          <tbody id="module-row<?php echo $module_row; ?>">
+            <tr>
+              <td class="left">
+                <select name="smaily_for_opencart_module[<?php echo $module_row; ?>][layout_id]">
+                  <?php foreach ($layouts as $layout) { ?>
+                  <?php if ($layout['layout_id'] == $module['layout_id']) { ?>
+                    <option value="<?php echo $layout['layout_id']; ?>" selected="selected"><?php echo $layout['name']; ?></option>
```

Comment:
Use the same pattern here to render select field options as you are using on lines 245-249 (with `position` select).